src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -116,9 +124,62 @@ attrsToMarkdown attribs = braces $ hsep [attribId, attribClasses, attribKeys]
               escAttrChar '\\' = literal "\\\\"
               escAttrChar c    = literal $ T.singleton c
 
+attrsToMarkua:: Attr -> Doc Text
+attrsToMarkua attributes
+    | null list = empty
+    | length list == 1 = braces $ head list
+    |  otherwise = braces $ hcat (head list : (map (\x -> ", " <> x)
```

Comment:
In general, it's better to avoid using `head` (since it's a partial function and can throw runtime errors).
Why not use a case?  Match on
``` hs
  [] -> empty
  [x] -> braces x
  xs -> braces $ intercalate ", " xs
```

Reply:
We changed it accordingly